tests/core/test_run_query_tools.py

```diff
@@ -0,0 +1,320 @@
+"""Unit tests for run query building with new TimeframeFilter and created_at support."""
+
+from datetime import datetime, timedelta, timezone
+
+from nominal_api import api
+
+from nominal.core._utils.query_tools import create_search_runs_query
+from nominal.ts import _SecondsNanos
+
+
+def test_create_search_runs_query_basic():
+    """Test basic query creation without filters."""
+    query = create_search_runs_query()
+
+    # Should return an empty AND query
+    assert hasattr(query, "and_")
+    assert query.and_ == []
```

Comment:
would be slightly better to not check exact equality for container types

```suggestion
    assert len(query.and_) == 0
```